projects/packages/waf/src/class-waf-runner.php

```diff
@@ -348,16 +362,22 @@ public static function create_blocklog_table() {
 	/**
 	 * Deactivates the WAF by deleting the relevant options and emptying rules file.
 	 *
+	 * @throws Waf_Exception         If the filesystem is unavailable.
+	 * @throws File_System_Exception If file writing fails.
+	 *
 	 * @return void
-	 * @throws \Exception If file writing fails.
 	 */
 	public static function deactivate() {
 		delete_option( self::MODE_OPTION_NAME );
 		delete_option( Waf_Rules_Manager::VERSION_OPTION_NAME );
 
 		global $wp_filesystem;
 
-		self::initialize_filesystem();
+		try {
+			self::initialize_filesystem();
+		} catch ( Waf_Exception $e ) {
+			throw $e;
+		}
```

Comment:
```suggestion
		self::initialize_filesystem();
```

🙂  I hope I am not missing something obvious with these.

Reply:
Resolved via https://github.com/Automattic/jetpack/commit/c1d3effd93c8fd23355781bca6b324383c18d498